### PR TITLE
Update with recent papers from Penn

### PR DESCRIPTION
**Pull request by alexzzhu**
Added cvpr19 and arxiv19, as well as links to code for icra17, cvpr17 and rss18 in algorithms.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -520,7 +521,8 @@ IEEE Conf. Computer Vision and Pattern Recognition (CVPR), 2019. [PDF](https://a
 IEEE Conf. Computer Vision and Pattern Recognition (CVPR), 2019.
 - [Gehrig et. al. arXiv 2019](#Gehrig19arxiv),  
 *End-to-End Learning of Representations for Asynchronous Event-Based Data*.
-
+- [Zhu et. al. CVPR 2019](#Zhu19cvpr),
```
Comment (guillermogb) [inline README.md]: This one is not needed. I wrote the paper in this same section, a few lines above.

File: README.md [hunk 2]
```diff
@@ -858,6 +861,11 @@ Int. Conf. Machine Vision Applications (MVA), 2019.
 - <a name="Afshar19arxiv"></a>Afshar, S., Xu, Y., Tapson, J., van Schaik, A., Cohen, G.,  
 *[Event-based Feature Extraction using Adaptive Selection Thresholds](https://arxiv.org/pdf/1907.07853)*,  
 arXiv:1907.07853, 2019
+- <a name="Zhu19arxiv"></a>Zhu, A.Z., Wang, Z., Daniilidis, K.,  
+*[Motion Equivariant Networks for Event Cameras with the Temporal Normalization Transform](https://arxiv.org/abs/1902.06820),* arXiv:1902.06820, 2019.
+- <a name="Zhu19cvpr"></a>Zhu, A.Z., Yuan, L., Chaney, K., Daniilidis, K.,  
+*[Unsupervised Event-Based Learning of Optical Flow, Depth, and Egomotion](http://openaccess.thecvf.com/content_CVPR_2019/papers/Zhu_Unsupervised_Event-Based_Learning_of_Optical_Flow_Depth_and_Egomotion_CVPR_2019_paper.pdf)*,
```
Comment (guillermogb) [inline README.md]: This CVPR should be written as in the ego-motion section (i.e., a link to the main entry).
There should be only one main entry for each paper; and currently there are two. Can you fix it?

File: README.md (hunk 1 above)
Comment (alexzzhu) [reply README.md]: Ah I didn't see that it'd been added already. I've removed my addition.

File: README.md (hunk 2 above)
Comment (alexzzhu) [reply README.md]: Done!

**Comment by guillermogb** [comment]
thanks

**Pull request merged**